slovak/marsmelaci.md

```diff
@@ -1,2 +1,6 @@
-Jedno nevšedné ráno sa zobudíš v divnej miestnosti, ktorá má steny z marshmallow na to, že máš v nose kyslú rybičku.
-
```

Comment:
Well, it's nice to have an empty line between the question "What do you do next" and the options. It will display better on GitHub that way.